### DISCUSSION TITLE
Feature Request: Read Strangers' Chat

### DISCUSSION THREAD

**Discussion by Nvrplygd** [thatskymod/Sky-CotL-Scripts#39] [Ideas]
Can i have lib who can read strangers chat again

**Comment by thatskymod** [comment]
Yes, **libLangit** is capable of reading chats from strangers.

You can download **libLangit** from https://github.com/thatskymod/Sky-CotL-Scripts/tree/main/Canvas/Libs.

The **"Read Chat"** feature can be accessed through "Main" -> "Witchcraft" -> "Read Chat".
